docs/book/how-to/setting-up-a-project-repository/stacks-pipelines-models.md

```diff
@@ -0,0 +1,85 @@
+- **Models**: [Models](../../how-to/use-the-model-control-plane/README.md) are entities that groups pipelines, artifacts, metadata, and other crucial business data together. You may think of a ZenML Model as a "project" or a "workspace" that spans multiple pipelines.
+- **Artifacts**: [Artifacts](../../user-guide/starter-guide/manage-artifacts.md) are the output of a pipeline step that you want to track and reuse across multiple pipelines.
+
+Understanding the relationships between stacks, pipelines, models, and artifacts is crucial for effective MLOps with ZenML.
+
+## How many Stacks do I need?
+
+A stack provides the infrastructure and tools for running pipelines. Think of a stack as a  representation of your execution environment in which your pipelines are run. This comprises both the hardware like the orchestration environment and any MLOps tools you use in your workflow. This way, Stacks allow you to seamlessly transition between different environments (e.g., local, staging, production) while keeping your pipeline code consistent.
+
+You can learn more about organizing and managing stacks in the [Managing Stacks and Components](../../how-to/stack-deployment/README.md) guide.
+
+You don't need a separate stack for each pipeline; instead, you can run multiple pipelines on the same stack. A stack is meant to be created once and then reused across multiple users and pipelines. This helps in the following ways:
+
+- reduces the overhead of configuring your infrastructure every time you run a pipeline.
+- provides a consistent environment for your pipelines to run in, promoting reproducibility.
+- reduces risk of errors when it comes to what hardware and tool configurations to use.
+
+As for Models, they are independent of the stack used to create them. Models in ZenML help you tie resources from multiple pipelines together, as you will learn in the next section.
+
+
+## How do I organize my Pipelines, Models, and Artifacts?
+
+Pipelines, Models, and Artifacts form the core of your ML workflow in ZenML. All of your project logic is organized around these concepts and as such, it helps to understand how they interact with each other and how to structure your code to make the most out of them.
+
+### Pipelines
+
+A pipeline typically encompasses the entire ML workflow, including data preparation, model training, and evaluation. It's a good practice to have a separate pipeline for different tasks like training and inference. This makes your pipelines more modular and easier to manage.
+
+- separation of pipelines by the nature of the task allows you to run them independently as needed. For example, you might train a model in a training pipeline only once a week but run inference on new data every day.
+- it becomes easier to manage and update your code as your project grows more complex.
+- different people can work on the code for the pipelines without interfering with each other.
+- it helps you organize your runs better.
```

Comment:
link to that run organization (develop locally) chapter?

Reply:
done